### ISSUE TITLE
Publishing JavaDocs to gh-pages on release  fails

### ISSUE THREAD

**Issue by heshamMassoud** [commercetools/commercetools-sync-java#118]
### Description
Publishing JavaDocs to gh-pages fails due to: 

````
:gitPublishReset FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':gitPublishReset'.
> org.eclipse.jgit.api.errors.TransportException: git@github.com:commercetools/commercetools-sync-java.git: Auth fail
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
````

**Comment by andrii-kovalenko-ct** [comment]
updated `GRGIT_USER` build variable. let's see if it helps.

**Comment by heshamMassoud** [comment]
closed by d1810eac76e661b32170d1c50c57348955e2a664
